livingdoc-engine/src/main/kotlin/org/livingdoc/engine/reporting/HtmlElements.kt

```diff
@@ -65,10 +66,13 @@ fun HtmlTable.rows(rows: List<RowResult>) {
     fun appendCellToDisplayFailedRowIfNecessary(newRow: Element, rowResult: Result) {
         if (rowResult is Result.Failed || rowResult is Result.Exception) {
             newRow.appendChild(
-                    Element("td").apply {
-                        setStyleClasses(HtmlReportTemplate.CSS_CLASS_BORDER_BLACK_ONEPX, determineCssClassForBackgroundColor(rowResult))
-                        appendChild(createFailedPopupLink(htmlTable.renderContext, rowResult))
-                    })
+                Element("td").apply {
+                    setStyleClasses(
+                        HtmlReportTemplate.CSS_CLASS_BORDER_BLACK_ONEPX,
```

Comment:
for better readability we could use parameter names here

Reply:
I don't see any nameable parameters here ;)